REMOTE_SETUP.MD

```diff
@@ -0,0 +1,37 @@
+1 step
```

Comment:
If it's supposed to be markdown I'd mark code parts as such, otherwise it looks quite ugly on GitHub.

Reply:
@sgeisler You can delete, these were work notes

Reply:
Will do. I guess I'll also do some squashing to avoid undoing changes that were done in previous commits, that might be a bit confusing for future readers otherwise.